### PR TITLE
Fizzbuzz with if, for done

### LINKED ISSUE #1: Do fizzbuzz with if, for
## Requirements

1부터 20까지 i가 1씩 증가하면서,
3의배수라면 fizz
5의배수라면 buzz
15의배수라면 fizzbuzz
그외의 모든 경우, 숫자 그대로를 출력한다.

## task list

- [x] print fizz
- [x] print buzz
- [x] print fizzbuzz

### PR DESCRIPTION
**Pull request by ulgoon**
close #1

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -1 +1,9 @@
-print('You can do this all day!')
+for i in range(1,20+1):
```
Comment (kingwangzzang1234) [inline fizzbuzz.py:1]: Increase endnum 20 to 30

File: fizzbuzz.py [hunk 2]
```diff
@@ -1 +1,9 @@
...
+    if i%15==0:
```
Comment (kingwangzzang1234) [inline fizzbuzz.py:2]: Could you do this with 1 if statement?

**Review by kingwangzzang1234 — CHANGES_REQUESTED** [review]

File: fizzbuzz.py (hunk 2 above)
Comment (ulgoon) [reply fizzbuzz.py:2]: Yes, sir!

**Review by kingwangzzang1234 — APPROVED** [review]
Well done!

**Pull request merged**